src/backupNg/backupNg.spec.js

```diff
@@ -30,5 +30,36 @@ describe("backupNg", () => {
       expect(omit(backupNg, "id", "userId")).toMatchSnapshot();
       expect(backupNg.userId).toBe(xo._user.id);
     });
+
+    it("creates a new backup job with schedules", async () => {
+      const { id: jobId } = await xo.createTempBackupNgJob({
+        ...defaultBackupNg,
+      });
+      const { id: scheduleId } = await xo.call("schedule.create", {
+        name: "scheduleTest",
+        jobId,
```

Comment:
No need to create it manually.  Simply pass the property `schedules` to the `backupNg.create`. 

Note: `schedules` contains items with fake id as key and the schedule as value and job's setting need to contain the fake schedule id with its retention.